lib/tasks/subscriptions/subscriptions.rake

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+namespace :ofn do
+  namespace :subs do
+    desc "Repeat placement job for a specific Order Cycle"
+    task repeat_placement_job: :environment do
+      puts "WARNING: this task will generate new, and potentially duplicate, orders for customers"
+
+      puts "Please input Order Cycle ID to reset"
+      input = STDIN.gets.chomp
```

Comment:
I personally would prefer a command line argument so that I don't have to re-type this.